helix-term/src/ui/prompt.rs

```diff
@@ -401,7 +401,7 @@ impl Prompt {
 
             let offset = self
                 .selection
-                .map(|selection| selection / items * items)
+                .map(|selection| selection / std::cmp::max(1, items * items))
```

Comment:
`/` and `*` are left-to-right associative in Rust but this is grouping them right-to-left: before it was `(selection / items) * items` but now it's `selection / (items * items)`.

Let's leave this expression untouched instead and check that `completion_area.height > 0` in the `if` condition for this block. Figuring out the offset doesn't make sense when there are no items and there's no need to try to render the completion popup if we don't have space for it